kiwi/repository/template/apt.py

```diff
@@ -29,6 +29,7 @@ def __init__(self) -> None:
             # kiwi generated apt-get config file
             Dir "/";
             Dir::State "${apt_shared_base}/";
+            Dir::State::status "/var/lib/dpkg/status.kiwi";
```

Comment:
As this reads, doesn't this mean we never use `/var/lib/dpkg/status` even for the image target inside? If so, then this is probably not the right way to do it. And does the `status.kiwi` file get included in the final image? If so, we probably don't want that.

Reply:
The way this works is that for the bootstrap phase an extra status file is used. This will *not end* up in the image. If the bootstrap phase is done and `apt-get` is called as chroot operation inside that bootstrapped root it starts basically from scratch with a fresh solver. This has a little glitch as it will queue packages from the bootstrap phase again in the install phase. As the exact same package has already been installed and downloaded the actual install of it does nothing but it's listed in the solver operation. Frankly speaking I prefer this way because the way we bootstrap is still a "frankenstein surgery"

Reply:
Excellent.